tests/operations/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -128,4 +144,25 @@ $a ?? $b;
 $this->reallyLong->testPropertyName =
     'a' . $someOtherTest . 'n' . $oneLastOneToMakeLineLong;
 
+'tesssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssssst' .
+    'test';
+
+$someReallyReallyReallyLongBooleanVariable +
+    $someReallyReallyReallyLongBooleanVariable;
+$someReallyReallyReallyLongBooleanVariable .
+    $someReallyReallyReallyLongBooleanVariable;
+$someReallyReallyReallyLongBooleanVariable .= $someReallyReallyReallyLongBooleanVariable;
+++$someReallyReallyReallyLongBooleanVariableWithReallyReallyReallyReallyReallyReallyLongName;
+--$someReallyReallyReallyLongBooleanVariableWithReallyReallyReallyReallyReallyReallyLongName;
+$someReallyReallyReallyLongBooleanVariableWithReallyReallyReallyReallyReallyReallyLongName++;
+$someReallyReallyReallyLongBooleanVariableWithReallyReallyReallyReallyReallyReallyLongName--;
+$someReallyReallyReallyLongBooleanVariable +
+    $someReallyReallyReallyLongBooleanVariable -
+    $someReallyReallyReallyLongBooleanVariable;
+
+$value = isset($_POST['menu-item-dropdown']) &&
```

Comment:
Related ?https://prettier.io/playground/#N4Igxg9gdgLgprEAuEASAbgQwDYFc4AEAvAQJYDO5cMAFKgPoAKA8gMoAqA2gOQC2CuALSl4vQQBMAThAAO4iAHco3ALoBKAgDJNAHSgEDZStTpM2XPgOGiJ0uYuUrOqflFwBJUQBEARu-HqeoYEAPwE3AiYPthw4txBhkjh4hRRMXEA3CAANCCyMKTQ5MigmJLSCoxlCMUomOgQpOI5IJjkMMgAZjhUuT6SmGAA1tSsMoOkUADmyDCS+LnyYF09cLmTVJIwjANTvJgr2L0gAFbkAB4AQgPDo5j8ADKTcIfHELgwMh8ATK9rIONJJtkCAfFEAJ7YaAtGSSSYwADqTRgAAtkAAOAAMuVhECoCIGMhBsLgm3QL1ykjgAEdcKQqTtMHsDkhukd-lReKRZvMOZMpjEAIq4CDwP65GBRJHiVHIb4SgakbD8gDCEF4+xBUGgFJAuCo7CitTZVAAvqagA

Reply:
hrmm interesting - ill try to take a look today